backend/inspirehep/records/serializers/bibtex.py

```diff
@@ -16,6 +20,25 @@
 
 
 class BibtexWriter(Writer):
+    math_expression_regex = re.compile(r"(?<!\\)\$.*?\$")
+
+    def _encode(self, text):
+        import latexcodec  # NOQA
+
+        if "$" not in text:
+            return codecs.encode(text, "ulatex+{}".format(self.encoding))
+        math_expressions = BibtexWriter.math_expression_regex.findall(text)
```

Comment:
attributes and methods on the current class can be accessed through `self`:

```suggestion
        math_expressions = self.math_expression_regex.findall(text)
```